source/dofs/dof_accessor_get.cc

```diff
@@ -50,7 +50,7 @@ DoFCellAccessor<DoFHandlerType, lda>::get_interpolated_dof_values(
   Vector<number> &   interpolated_values,
   const unsigned int fe_index) const
 {
-  if (!this->has_children())
```

Comment:
I would just keep it here.

Reply:
I adjusted the text, but I think that the condition we want to test here really is whether the cell is active, because that is when data is available on it.